conformance/tests/specialtypes_promotions.py

```diff
@@ -2,6 +2,8 @@
 Tests "type promotions" for float and complex when they appear in annotations.
 """
 
+from typing import assert_type
```

Comment:
It appears this import is no longer necessary.

Reply:
No, there is still an `assert_type()` call (line 20).

Reply:
Never mind! :)